site-src/guides/implementers.md

```diff
@@ -159,16 +159,118 @@ the `metadata.generation` field of the object at the time the status is generate
 This allows users of the API to determine if the status is relevant to the current
 version of the object.
 
-
-### Resource details
+## TLS
+
+TLS is a large topic in Gateway API, with the set of capabilities continuing to
+expand. There is a [TLS guide](/guides/tls) that covers this topic in more depth
+from a user-facing perspective, but this section attempts to fill in some gaps
+from an implementer's perspective.
+
+### Listener Isolation
+Within Gateways, TLS config is currently tied exclusively to Listeners. To make
+that approach manageable, we're encouraging all implementations to work towards
+the goal of providing full "Listener Isolation" as defined below:
+
+Requests SHOULD match at most one Listener. For example, if Listeners are
+defined for "foo.example.com" and "*.example.com", a request to
+"foo.example.com" SHOULD only be routed using routes attached to the
+"foo.example.com" Listener (and not the "*.example.com" Listener).
+
+Implementations that do not support Listener Isolation MUST clearly document
+this. In the future, we plan to add Listener Isolation conformance tests to
+ensure this behavior is consistent across implementations that claim support for
+the feature.
+
+### Indirect Configuration
+There are a variety of instances where TLS Certificates may not be managed
+directly by the owner of a Gateway. Although this is not meant to be an
+exhaustive list, it documents some of the approaches we expect to see used to
+manage TLS Certificates with Gateway API:
+
+#### 1. Certificates from other places
+Some providers offer the ability to configure and host TLS Certificates outside
+of Kubernetes altogether. Implementations that can connect to those external
+providers may expose that capability via a TLS option on the Gateway Listener,
+for example:
+
+```
+  listeners:
+  - name: https
+    protocol: HTTPS
+    port: 443
+    tls:
+      mode: Terminate
+      options:
+        vendor.example.com/certificate-name: store-example-com
+```
+
+In this example, the `store-example-com` name would refer to the name of a
+certificate stored by the external `vendor.example.com` TLS Certificate
+provider.
+
+#### 2. Automatically generated TLS certs that are populated later
+Many users would prefer that TLS certs were automatically generated on their
```

Comment:
```suggestion
Many users would prefer that TLS certs be automatically generated on their
```

Reply:
For some reason "be" doesn't feel quite right to me here, but I'm not exactly a grammar expert. Maybe there's a better way to phrase the whole sentence?

Reply:
It feels a little more readable to me to use `will be` in place of `be`.

But, I think it depends on where we're talking about this behavior happening.

- `were`, in the past
- `are` in the present
- `will be` in the future.

Personally I think that `will be` is probably what we're going for.

Reply:
Thanks! Updated to "will be".